src/cmd/src/standalone.rs

```diff
@@ -184,12 +191,31 @@ impl StartCommand {
 /// Build frontend instance in standalone mode
 async fn build_frontend(
     fe_opts: FrontendOptions,
+    plugins: AnyMap,
     datanode_instance: InstanceRef,
 ) -> Result<Frontend<FeInstance>> {
     let mut frontend_instance = FeInstance::new_standalone(datanode_instance.clone());
     frontend_instance.set_catalog_manager(datanode_instance.catalog_manager().clone());
     frontend_instance.set_script_handler(datanode_instance);
-    Ok(Frontend::new(fe_opts, frontend_instance))
+    Ok(Frontend::new(fe_opts, frontend_instance, plugins))
+}
+
+impl TryFrom<&StartCommand> for AnyMap {
```

Comment:
Since the two `StartCommand` are different, it would be hard to extract the logic elsewhere directly. 

Another way to deal with it would be explicitly passing all config value from `StartCommand`, so the struct itself is not used in the function signature. e.g:
```
fn load_frontend_plugins(
    user_provider: Option<String>,
    other_option: Option<String>,
) -> Result<AnyMap> {
    // ...
}
```
then the `load_frontend_plugins` function can be placed in `frontend` module.

Reply:
Change to a unified function. PTAL
```
pub fn load_frontend_plugins(user_provider: &Option<String>) -> Result<AnyMap> {
    let mut plugins = AnyMap::new();

    if let Some(provider) = user_provider {
        let provider = auth::user_provider_from_option(provider).context(IllegalAuthConfigSnafu)?;
        plugins.insert::<UserProviderRef>(provider);
    }
    Ok(plugins)
}
```